docs/develop/sdks-integrations/deploying-javascript-web-sdk.md

```diff
@@ -0,0 +1,89 @@
+---
+sidebar_position: 5
+sidebar_class_name: sidebar-item-websdk
+sidebar_label: Web SDK
+title: Momento Web SDK for JavaScript in browsers
+description: Get up and running quickly with the Javascript web-sdk for use in browsers connecting to a Momento Cache or Topics services.
+---
+
+# Momento Web SDK for Javascript in browsers
+
+Momento provides two JavaScript SDKs; one for Node.js and one for other web applications.  The two SDKs have
+identical APIs, so your code will look the same except for `import` statements, but under the hood they are
+built for optimal performance and compatibility in different JavaScript runtime environments.
+
+The Node.js SDK is best suited for server-side use cases. The Momento Web SDK, however, allows developers to write
+JavaScript code that runs in a browser and communicates directly with Momento services. This allows you to avoid the
+typical overhead of building and operating your own web service to mediate cache or pub/sub calls between the browser
+and Momento.  It also means one less hop for your web traffic, so you can get even performance out of your browser
+application.  The best of both worlds1
```

Comment:
Typo - there's a 1 instead of an exclamation point

Reply:
Looks like there are carriage returns where there shouldn't be. I will fix these all really quickly

Reply:
I also have to fix the double spaces after sentences by @cprice404. for shame!!! ;-P

Reply:
Fixed all of this

Reply:
lol I really want to train my brain to stop with the double spaces but it's just so ingrained :)

Reply:
No excuses!! hehehe I had to retrain my brain to stop doing it as well. I believe in you!